src/js/components/Home/Map.js

```diff
@@ -8,8 +8,13 @@ import {
 import { Motion, spring } from "react-motion";
 
 const mapStyles = {
-    width: "50%",       // Where the globe width size on the page is set.
-    height: "auto"
+<<<<<<< HEAD
```

Comment:
You pushed Merge Conflict

Reply:
My eyes

Reply:
Fixed, was a conflict and miss communication that's how it happened, apologies for the inconvience.

Reply:
I dont see it fixed?